pkgs/development/python-modules/decora-wifi/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, python313Packages }:
+
+buildPythonPackage rec {
+  pname = "decora_wifi";
+  version = "1.4";
+
+  src = fetchPypi {
```

Comment:
No tag on Github (https://github.com/tlyakhov/python-decora_wifi), so `fetchPypi` is OK to use.